src/services/dumpers/agent-nodejs-dumper.ts

```diff
@@ -0,0 +1,176 @@
+import type { ConfigInterface, DbConfigInterface } from '../../interfaces/project-create-interface';
+
+import _ from 'lodash';
+
+import toValidPackageName from '../../utils/to-valid-package-name';
+import AbstractDumper from './abstract-dumper';
+
+export default class AgentNodeJsDumper extends AbstractDumper {
+  private env: { FOREST_SERVER_URL: string; FOREST_URL_IS_DEFAULT: string };
+
+  private readonly DEFAULT_PORT = 3310;
+
+  private readonly isLinuxOs: boolean;
+
+  constructor(context) {
+    const { assertPresent, env, isLinuxOs } = context;
+
+    assertPresent({
+      env,
+      isLinuxOs,
+    });
+
+    super(context);
+
+    this.env = env;
+    this.isLinuxOs = isLinuxOs;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  protected get templateFolder() {
+    return 'agent-nodejs/';
+  }
+
+  writePackageJson(dbDialect: string, applicationName: string) {
+    const dependencies: { [name: string]: string } = {
+      dotenv: '^16.0.1',
+      '@forestadmin/agent': '^1.0.0',
+    };
+
+    if (dbDialect === 'mongodb') {
+      dependencies['@forestadmin/datasource-mongoose'] = '^1.0.0';
+    } else {
+      dependencies['@forestadmin/datasource-sql'] = '^1.0.0';
+    }
+
+    if (dbDialect) {
+      if (dbDialect.includes('postgres')) {
+        dependencies.pg = '~8.2.2';
+      } else if (dbDialect === 'mysql') {
+        dependencies.mysql2 = '~2.2.5';
+      } else if (dbDialect === 'mssql') {
+        dependencies.tedious = '^6.4.0';
+      }
+    }
+
+    const pkg = {
+      name: toValidPackageName(applicationName),
+      version: '0.0.1',
+      private: true,
+      main: 'index.js',
+      scripts: {
+        'start:watch': 'nodemon',
+        start: 'node ./index.js',
+      },
+      nodemonConfig: {
+        ignore: ['./forestadmin-schema.json'],
+      },
+      dependencies,
+      devDependencies: {
+        nodemon: '^2.0.12',
+      },
+    };
+
+    this.writeFile('package.json', `${JSON.stringify(pkg, null, 2)}\n`);
+  }
+
+  writeIndex(dbDialect: string) {
+    const isMongoose = dbDialect === 'mongodb';
+
+    const context = {
+      isMongoose,
```

Comment:
The more we test in unit tests, the better I feel. 

Like that we are forced to assert that the datasource imported is the actual one in integration tests. 

Lets keep the datasourceimport paramater